code/modules/mob/animations.dm

```diff
@@ -315,16 +315,16 @@ note dizziness decrements automatically in the mob's Life() proc.
 
 /atom/movable/proc/do_putdown_animation(atom/target, mob/user)
 	spawn()
-		if (!istype(src) || QDELETED(src))
+		if (QDELETED(src))
 			return
-		if (!istype(target) || QDELETED(target))
```

Comment:
same here, the istype exists for literally checking the type.

If you don't specify a type in the proc call, it implicitly uses the type the variable is declared with

istype(src) is bullshit though

Reply:
Same as above.